Jenkinsfile

```diff
@@ -24,7 +24,7 @@ pipeline {
         stage('Unit Test Python')      { steps { sh 'make TEST_CONCRETE_BACKEND=llvm unittest-python' } }
         stage('Mandos Unit Test')      { steps { sh 'make TEST_CONCRETE_BACKEND=llvm mandos-test -j4' } }
         stage('Adder Contract Test')   { steps { sh 'make TEST_CONCRETE_BACKEND=llvm elrond-adder-test' } }
-        stage('Lottery Contract Test') { steps { sh 'make TEST_CONCRETE_BACKEND=llvm elrond-lottery-test' } }
+//      stage('Lottery Contract Test') { steps { sh 'make TEST_CONCRETE_BACKEND=llvm elrond-lottery-test' } }
```

Comment:
Please just remove it, better to not have commented out parts in our scripts. You can add an issue if you want.